ledger/participant-state/kvutils/src/test/lib/scala/com/daml/ledger/participant/state/kvutils/SimplePackage.scala

```diff
@@ -114,25 +48,40 @@ class SimplePackage(additionalContractDataType: String) {
       templateId: Ref.Identifier,
       choiceName: Ref.ChoiceName,
   ): Command =
-    ExerciseCommand(templateId, contractId, choiceName, ValueUnit)
+    ExerciseCommand(
+      templateId,
+      contractId,
+      choiceName,
+      ValueRecord(Some(typeConstructorId(s"Simple:$choiceName")), ImmArray.empty),
```

Comment:
ValueUnit doesn't work anymore. It took me a while to work around it.

Reply:
Maybe worth factoring out the choice argument given that you reuse it everywhere?